docs/organization/dynamic-sampling/index.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Dynamic Sampling
+sidebar_order: 30
+description: "Learn how to prioritize important events and increase visibility in lower-volume projects with Dynamic Sampling."
+---
+
+## Overview
+
+Projects that generate many events can quickly consume a large portion of your billing quota. This may create an imbalance where high-volume projects get more visibility, leaving insufficient quota for monitoring your other projects. That's where dynamic sampling comes in.
+
+Dynamic Sampling employs advanced sampling techniques to retain a representative sample of the data you send. Its algorithms are designed to prioritize keeping samples that are valuable for debugging and analysis, while reducing the retention of common or redundant data.
+
+This process is applied directly on Sentry's servers, which allows you to configure and adjust sampling rates for stored spans right from the UI without needing to modify your SDK. This makes it possible for you to make instant updates without waiting for code freezes, app store approvals, or redeployments.
+
+By analyzing incoming traffic patterns, Dynamic Sampling is able to tailor its decisions to both the actual traffic volume as well as the content of accepted spans. For more details, check out the [Dynamic Sampling Priorities](#dynamic-sampling-priorities) section.
+
+<Alert>
+  Dynamic Sampling is available for customers on our latest Enterprise plans,
+  starting June, 2024.
+</Alert>
+
+## Prerequisites
+
+- [Admin-level permissions](/organization/membership/).
+- Latest version of one of the below SDKs:
+  - Python: 1.7.2 or later
+  - JavaScript: 7.6.0 or later
+  - Apple: 7.23.0 or later
+  - Android: 6.5.0 or later
+  - React Native: 4.3.0 or later
+  - Dart and Flutter: 6.11.0 or later
+  - PHP: 3.9.0 or later
+  - Laravel: 3.0.0 or later
+  - Symfony: 4.4.0 or later
+  - Ruby: 5.5.0 or later
+  - Java: 6.5.0 or later
+  - .NET: 3.22.0 or later
+  - Go: 0.16.0 or later
+
+## Configuration
+
+In this section you'll learn how to use Dynamic Sampling in your organization.
+
+![The dynamic sampling mode and target sample rate setting.](./img/mode-and-target-sample-rate.png)
+
+### Modes
+
+If you prefer not to set sample rates manually, Sentry can do it for you. You just have to set a global sample rate for your organization and Sentry will then automatically balance the sample rates across projects, giving priority to factors like low-volume endpoints and new releases. If you want more fine-grained control, you can customize data retention by setting individual sample rates for specific projects.
```

Comment:
Here I miss an introductory sentence, something like "we offer two modes: simple mode and advanced mode".